travis-ymls/jicofo.travis.yml

```diff
@@ -0,0 +1,33 @@
+# ----------------------------------------------------------------------------
+#
+# Package             : jicofo
+# Source Repo         : https://github.com/jitsi/jicofo
+# Travis Job Link     : https://travis-ci.com/github/asellappen/jicofo/
+# Created travis.yml  : Yes
+# Maintainer          : Arumugam N S <asellappen@yahoo.com>/Priya Seth<sethp@us.ibm.com>
+#
+# Script License      : Apache License, Version 2 or later
+#
+# ----------------------------------------------------------------------------
+language: java
+
+jdk:
+  - openjdk11
+  - openjdk15
+  
+sudo: True
+
+dist : focal
+
+arch:
+  - amd64
+  - ppc64le
+
+before_install: 
+  - mkdir -p /opt/maven
+  - curl https://downloads.apache.org/maven/maven-3/3.6.3/binaries/apache-maven-3.6.3-bin.tar.gz|tar -xz --strip 1 -C /opt/maven
```

Comment:
What is the reason for installing mvn explicitly ? It should be automatically there in travis right ?